service/controller/v26/resource/configmap/current.go

```diff
@@ -31,7 +33,12 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 	var currentConfigMaps []*apiv1.ConfigMap
 	{
 		namespace := key.ClusterNamespace(customResource)
-		configMapList, err := r.k8sClient.CoreV1().ConfigMaps(namespace).List(apismetav1.ListOptions{})
+
+		listOptions := apismetav1.ListOptions{
```

Comment:
```suggestion
		lo := apismetav1.ListOptions{
```

Not a blocker but I think this can be shorter.